pkg/scheduler/controller.go

```diff
@@ -49,13 +47,24 @@ const (
 	ControllerName = "edge-scheduler"
 )
 
-// NewController returns a new controller placing namespaces onto locations by create
-// a placement annotation..
+type eventSource string
+
+const (
+	eventSourceEdgePlacement eventSource = "EdgePlacement"
+	eventSourceLocation      eventSource = "Location"
+	eventSourceSyncTarget    eventSource = "SyncTarget"
+)
+
+type queueItem struct {
+	key         string
+	eventSource eventSource
```

Comment:
I think the most important thing about this field is that it indicates the kind of object being referenced.

Reply:
Renamed this.